dev/src/main/resources/META-INF/plugin.xml

```diff
@@ -14,8 +14,50 @@
     <vendor email="codewind-dev@eclipse.org" url="https://www.eclipse.org/codewind/">Codewind</vendor>
 
     <description><![CDATA[
-        Adds support for developing cloud-native, containerized web applications.
-    ]]>    </description>
+      <h2>Codewind</h2>
+      <p>
+        Adds tools to help build cloud-native, containerized Java
+        applications. Create an application from a template project
+        and launch, update, test, and debug in containers on the
+        desktop. Supports Codewind version 0.10.0. For more
```

Comment:
Maybe say, "0.10.0 and onward"?

Reply:
At this point the 0.10.0 version of the plugin only supports the 0.10.0 version of Codewind (this is true for all the IDEs). Until that changes, I think we'll have to update the supported version every time we release.

Reply:
Oh, OK, so this text is appearing specifically for the 0.10.0 plugin. I had thought it was appearing across any and all versions. If the text is version-specific, I could see why we'd want to leave the text as it is with mention of 0.10.0. Thanks!